rest-api/src/test/scala/za/co/absa/enceladus/rest_api/package.scala

```diff
@@ -15,7 +15,7 @@
 
 package za.co.absa.enceladus
 
-package object menas {
+package object restapi {
```

Comment:
I could not name this package object in tests to rest_api. Hopefully it's not too confusing, given that the module is rest-api and the package is rest_api

Reply:
I consider this acceptable.